server/src/inmantals/server.py

```diff
@@ -144,11 +145,13 @@ def treeify_reverse(iterator):
                     ],
                 )
             await self.publish_diagnostics(params)
-            logger.exception("Compile failed")
+            await self.send_show_message(1, "Compilation failed: " + e.get_message())
```

Comment:
If `params` is set, vscode shows the error location in the file and lists it under "problems" (the diagnostics call). Showing a message in that case would be somewhat obsolete and it might draw attention away from the more informative diagnostics. On the other hand it would be consistent. I'm not sure which would be best. @andraskvr wdyt?
```suggestion
            if params is None:
                await self.send_show_message(1, "Compilation failed: " + e.get_message())
```

Reply:
Ideally I would have liked for the message to be clickable, and to redirect toward the "problems" panel, but I couldn't find a way.